packages/cli/fragments/fragment-leptos/src/app.rs

```diff
@@ -60,7 +60,10 @@ pub fn App(cx: Scope) -> impl IntoView {
                 <a href="https://github.com/rust-lang/rust-analyzer" target="_blank">"rust-analyzer"</a>
             </p>
 
-            <form class="row" on:submit=greet>
+            <form class="row" on:submit=move |ev: SubmitEvent| {
+                ev.prevent_default()
+                greet()
+            }>
```

Comment:
is there a reason behind this change? I think it should be reverted to have consistency with other templates and just call `ev.prevent_default()` above inside the closure

Reply:
Ummm... I don't know why I didn't think of that... yeah that makes more sense... :|